frontend/pages/sign-up.tsx

```diff
@@ -0,0 +1,299 @@
+              register={register}
+              registerOptions={{
+                required: intl.formatMessage({
+                  defaultMessage: 'You need to enter your email',
+                  id: 'NioTuC',
+                }),
+                validate: (value) =>
+                  validate(
+                    { from: value },
+                    {
+                      from: {
+                        email: true,
+                      },
+                    }
+                  ),
+              }}
+            />
+          </label>
+          {errors.email && (
+            <p id="email-error" className="mt-1 ml-2 font-sans text-xs text-red font-regular">
+              {errors.email.message || (
+                <FormattedMessage defaultMessage="Invalid email format" id="nc1IrM" />
+              )}
+            </p>
+          )}
+        </div>
+        <div className="flex gap-4">
+          <div className="w-full">
+            <label htmlFor="password">
+              <p className="mb-2.5 mt-4.5 font-sans text-sm font-semibold text-gray-800">
+                <FormattedMessage defaultMessage="Password" id="5sg7KC" />
+              </p>
+              <Input
+                type="password"
+                placeholder="insert password"
+                name="password"
+                id="password"
+                aria-describedby="password-description password-error"
+                register={register}
+                registerOptions={{
+                  required: intl.formatMessage({
+                    defaultMessage: 'You need to enter a password.',
+                    id: 'zeCjLr',
+                  }),
+                  minLength: {
+                    value: 8,
+                    message: intl.formatMessage({
+                      defaultMessage: 'The password must have at least 8 characters.',
+                      id: 'TiXJ+4',
+                    }),
+                  },
+                }}
+              />
+            </label>
+            <p
+              className="mt-1 font-sans text-xs text-gray-400 font-regular"
+              id="password-description"
+            >
+              <FormattedMessage defaultMessage="Use at least 8 characters." id="BvrO01" />
+            </p>
+            {errors.password && (
+              <p id="password-error" className="mt-1 ml-2 font-sans text-xs text-red font-regular">
+                {errors.password.message}
+              </p>
+            )}
+          </div>
+          <div className="w-full">
+            <label htmlFor="confirm-password">
+              <p className="mb-2.5 mt-4.5 font-sans text-sm font-semibold text-gray-800">
+                <FormattedMessage defaultMessage="Confirm password" id="8HimVK" />
+              </p>
+              <Input
+                type="password"
+                id="confirm-password"
+                name="confirmPassword"
+                placeholder="insert password"
```

Comment:
Needs to be translated. For consistency with the other placeholder, you should also capitalise: "Insert password".